src/main/java/frc/robot/util/AutoGenerator.java

```diff
@@ -72,6 +81,7 @@ public Command getAutoCommand() {
     public Command getExitAndBalance(StartLocation startArea) {
         if(startArea == StartLocation.ONE || startArea == StartLocation.TWO) {
             PathPlannerTrajectory path = PathPlanner.loadPath("ExitAndBalance A" + startArea.value, defaulPathConstraints);
+            eventMap.put("scoreConeHigh", new ExtendAndScore(drivetrainSubsystem, armSubsystem, intake));
             eventMap.put("autoBalance", new PrintCommand("Auto balance begin"));
```

Comment:
I think ideally we would create the eventMap and its entries before passing it to the builder in the constructor. I was thinking these would be constant across all our auto commands and we could essentially reuse the event names